MoreLinq/AtMost.cs

```diff
@@ -0,0 +1,64 @@
+#region License and Terms
+// MoreLINQ - Extensions to LINQ to Objects
+// Copyright (c) 2015 sholland. All rights reserved.
```

Comment:
The copyright notices should correspond to the author. See 57295fff782ddc22d57c3883ed70dd77bd06e338.